### PR TITLE
:closed_lock_with_key: Set Azure to redirect all traffic to https

### LINKED ISSUE #146: Redirect http traffic to https
Ensure all traffic if redirected to `https`

### PR DESCRIPTION
**Pull request by st3v3nhunt**
Setting up environment in Azure to test the change before merging where it will be automatically deployed to the staging slot.

New slot added and pointing to this branch i.e. `feature/https-all-traffic`
http://connecting-to-services-steve-test.azurewebsites.net/

The site automatically redirects to https://connecting-to-services-steve-test.azurewebsites.net/finders/find-help

Fixes #146

### CODE REVIEW

File: web.config [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+     https://github.com/tjanczuk/iisnode/blob/master/src/samples/configuration/web.config
+-->
+
+<configuration>
+  <system.webServer>
+    <!-- Visit http://blogs.msdn.com/b/windowsazure/archive/2013/11/14/introduction-to-websockets-on-windows-azure-web-sites.aspx for more information on WebSocket support -->
+    <webSocket enabled="false" />
+    <handlers>
+      <!-- Indicates that the server.js file is a node.js site to be handled by the iisnode module -->
```
Comment (neilbmclaughlin) [inline web.config]: Comment is incorrect. Correct or, preferably, delete.

**Comment by st3v3nhunt** [comment]
The file is the autogenerated one with the addition of the rule. I'll make it clearer.

**Comment by st3v3nhunt** [comment]
Do you want it with comments or without?

**Comment by neilbmclaughlin** [comment]
Ah, I did wonder about if it might be :)

Not worth doing it now but in a similar situation would be worth adding as one commit and updating as another.

**Comment by st3v3nhunt** [comment]
It is easy enough to do and probably a good idea

**Comment by neilbmclaughlin** [comment]
Regarding comments - I don't think this comment adds anything.

**Comment by st3v3nhunt** [comment]
Do you want a fresh auto generated version or one without that comment? Or to remove that comment later?

**Comment by neilbmclaughlin** [comment]
Assuming the autogenerated one already contains the erroneous comment I'd commit exactly as it is with the error. The 2nd commit for the http redirection could then include the removal of the comment with a commit note.

**Comment by st3v3nhunt** [comment]
Fine grained changes detailed through commit history as requested

**Pull request merged**